packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -422,21 +476,31 @@ Array [
         'darwin-arm64',
         'debian-openssl-1.0.x',
         'debian-openssl-1.1.x',
+        'debian-openssl-3.0.x',
         'linux-arm64-openssl-1.0.x',
         'linux-arm64-openssl-1.1.x',
+        'linux-arm64-openssl-3.0.x',
         'rhel-openssl-1.0.x',
         'rhel-openssl-1.1.x',
+        'rhel-openssl-3.0.x',
         'windows',
         'linux-musl',
       ],
       version: FIXED_BINARIES_HASH,
     })
     const after = Date.now()
+    // TODO: the comment below says 2s, but the test checks for 20s.
+    // The numbers in the comments are also way below the actual time.
+    //
     // cache should take less than 2s
     // value on Mac: 1440
     // value on GH Actions: ~5812
     const took = after - before
     expect(took).toBeLessThan(20000)
```

Comment:
I think this was to avoid flakiness, while this can be fast on a dedicated machine, in CI it's slower and varies.